packages/core/core.ts

```diff
@@ -84,18 +93,37 @@ export async function initFuzzing(options: Options) {
 				: new MemorySyncIdStrategy()
 		)
 	);
+	// Loads custom hook files and adds them to the hook manager.
```

Comment:
Could the hook/bug detector filtering happen here if the build-in bug detectors are added to the hook list?

Reply:
Unfortunately, `importModule` doesn't get triggered when built-in modules are imported. Also a bug detector might hook several functions at the same time, or start a server without hooking anything. This would still require a guard to prevent loading the bug detector, which we now have with --disable-bug-detectors="regex".

Reply:
There's probably a misunderstanding. I meant to somehow add the internal bug detector files to `options.customHooks`, so that they would be imported here, or not, depending on a filter criteria. So basically what `loadBugDetectors` currently does. WDYT?

Reply:
will proceed as discussed